src/plugins/api-proxy/util/duotone.js

```diff
@@ -70,6 +70,16 @@ export const duotones = [
  * @module duotoneServer
  */
 
+export const makeSign = (salt, ref) => rx => {
+	const spec = `event/${rx}/${ref}`;
+	const signature = crypto
+		.createHash('sha1')
+		.update(`${spec}${salt}`)
+		.digest('hex')
+		.substring(0, 10);
+	return `https://a248.e.akamai.net/secure.meetupstatic.com/photo_api/${spec}/sg${signature}`;
```

Comment:
👍 Yeah Lori opened a ticket about that for me, so it's updated in `master` and I'll make sure the merges bring the update here